objects/src/transaction/tx_witness.rs

```diff
@@ -0,0 +1,147 @@
+use super::{
+    AccountId, AdviceInputs, ConsumedNoteInfo, Digest, Felt, Hasher, Program, StackInputs,
+    StarkField, TransactionWitnessError, Word, WORD_SIZE,
+};
+
+/// A [TransactionWitness] is the minimum required data required to execute and prove a Miden rollup
+/// transaction.
+///
+/// The [TransactionWitness] is composed of:
+/// - account_id: the account id of the account the transaction is being executed against.
+/// - initial_account_hash: the initial account hash of the account the transaction is being
+///   executed against.
```

Comment:
minor nit: we can probably just say "the hash of the initial state of the account the transaction is being executed against."